src/components/styles/index.ts

```diff
@@ -1,6 +1,6 @@
 import { Divider } from "./Divider";
```

Comment:
This file should be lower case and doesn't need the TSX extension. `Divider.tsx`